packages/nextjs/app/token-vendor/page.tsx

```diff
@@ -25,40 +25,42 @@ const TokenVendor: NextPage = () => {
   const [tokensToSell, setTokensToSell] = useState<string>("");
 
   const { address: connectedAddress } = useAccount();
-
-  // ToDo: Add YourToken symbol
+  const { data: yourTokenSymbol } = useScaffoldReadContract({
+    contractName: "YourToken",
+    functionName: "symbol",
+  });
 
   const { data: yourTokenBalance } = useScaffoldReadContract({
     contractName: "YourToken",
     functionName: "balance_of",
     args: [connectedAddress ?? ""],
   });
 
+  const { data: tokensPerEth } = useScaffoldReadContract({
+    contractName: "Vendor",
+    functionName: "tokens_per_eth",
+  });
+
+  const { writeAsync: transferTokens } = useScaffoldWriteContract({
+    contractName: "YourToken",
+    functionName: "transfer",
+    args: [toAddress, multiplyTo1e18(tokensToSend)],
+  });
+
+  // // Vendor Balances
   const { data: vendorContractData } = useDeployedContractInfo("Vendor");
 
   const { data: vendorTokenBalance } = useScaffoldReadContract({
     contractName: "YourToken",
     functionName: "balance_of",
     args: [vendorContractData?.address ?? ""],
+    watch: true,
```

Comment:
this watch:true is not necessary

Reply:
perfect remove, and add pending in vendorContractBalance in last commit